src/registry/warp-utils.ts

```diff
@@ -19,3 +24,60 @@ function getWarpAddressKey(standard: TokenStandard): string | null {
   if (standardValue.includes('native')) return 'native';
   else return null;
 }
+
+/**
+ * Gets a warp route ID from a warp route config path.
+ * @param configRelativePath A relative path in the deployments dir
+ *    (e.g. `warp_routes/USDC/ethereum-arbitrum-config.yaml`)
+ */
+export function warpRouteConfigPathToId(configRelativePath: string): WarpRouteId {
+  const matches = configRelativePath.match(WARP_ROUTE_CONFIG_FILE_REGEX);
+  if (!matches || matches.length < 3)
+    throw new Error(`Invalid warp route config path: ${configRelativePath}`);
+  const [_, tokenSymbol, chains] = matches;
+  return createWarpRouteConfigId(tokenSymbol, chains.split('-'));
+}
+
+/**
+ * Gets a warp route ID from a warp route config.
+ */
+export function warpRouteConfigToId(config: WarpCoreConfig): WarpRouteId {
+  if (!config?.tokens?.length) throw new Error('Cannot generate ID for empty warp config');
+  const tokenSymbol = config.tokens[0].symbol;
```

Comment:
what if there are multiple token symbols?

Reply:
Considered that, this would just take the first. Supporting different symbols in warp configs would imply our current symbol-based file organization would need to change. Decided not to pull that thread atm

Reply:
Added info to the fn comment header to clarify